zkevm-circuits/src/evm_circuit/execution/end_tx.rs

```diff
@@ -256,7 +256,7 @@ impl<F: Field> ExecutionGadget<F> for EndTxGadget<F> {
         } else {
             let rw = &block.rws[(
                 RwTableTag::TxReceipt,
-                (tx.id - 1) * TxReceiptFieldTag::COUNT - 1,
+                (tx.id - 1) * (TxReceiptFieldTag::COUNT + 1) - 2,
```

Comment:
should be: 
` let idx = (tx.id - 2) * (TxReceiptFieldTag::COUNT + 1) + 2;` as earlier discussion ?